common/app/common/Edition.scala

```diff
@@ -111,10 +111,10 @@ object Edition {
     val path = request.path
     path match {
       case EditionalisedId(editionId, section) if Edition.defaultEdition.isEditionalised(section.drop(1)) =>
-        val links = Edition.editionsByRequest(request).map(EditionLink(_, section))
+        val links = Edition.all.map(EditionLink(_, section))
         links.filter(link => link.edition.isEditionalised(link.path.drop(1)))
       case EditionalisedFront(_) =>
-        editionsByRequest(request).map(EditionLink(_, "/"))
+        Edition.all.map(EditionLink(_, "/"))
```

Comment:
I think this is a little fragile, because we're not planning to exclude 'Europe' from the list of alternates because it's a _beta_ edition- we're still planning to exclude it even when it's _not_ beta, and is included in `Edition.all`!

I think maybe it would be better to make the 'locale' field on Edition optional?